apps/api/src/app/workflows-v2/exceptions/step-upsert-mechanism-failed-missing-id.exception.ts

```diff
@@ -0,0 +1,13 @@
+import { InternalServerErrorException } from '@nestjs/common';
+import { NotificationStepEntity } from '@novu/dal';
+
+export class StepUpsertMechanismFailedMissingIdException extends InternalServerErrorException {
+  constructor(stepDatabaseId: string, stepExternalID: string | undefined, persistedStep: NotificationStepEntity) {
+    super({
+      message: 'Failed to upsert step control values due to missing id',
```

Comment:
Please add the database ids and external ids to the error message.

```suggestion
      message: 'Failed to upsert step control values due to missing id',
```

Reply:
it's a part of the error object, I'm not adding to the actual msg on purpose